examples/minioinstance.yaml

```diff
@@ -114,18 +114,20 @@ spec:
   ## is not working properly and needs restart. Kubernetes automatically
   ## restarts the pods if liveness checks fail.
   liveness:
-    initialDelaySeconds: 120
-    periodSeconds: 15
-    timeoutSeconds: 12 # should always greater than MINIO_API_READY_DEADLINE (which defaults to 10s)
+    initialDelaySeconds: 35
+    periodSeconds: 5
+    failureThreshold: 1000
+    timeoutSeconds: 60 # should always greater than MINIO_API_READY_DEADLINE (which defaults to 10s)
```

Comment:
This comment is for readiness only not liveness @kerneltime :-) it was a mistake from previous PR

Reply:
Will fix it :-)